index.bs

```diff
@@ -2789,6 +2789,31 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats for Packed Attestation, FIDO U2F Attestation, and Assertion Signatures ### {#signature-attestation-types}
```

Comment:
I would more clearly explain that with respect to [=assertion signatures=] that this is for assertions produced using the three listed signature algorithms.

what about any of the other COSE sig algorithms ?

Reply:
They will refer to COSE document.